public/app/plugins/datasource/loki/queryUtils.ts

```diff
@@ -304,3 +305,12 @@ export function requestSupportsSplitting(allQueries: LokiQuery[]) {
 
   return queries.length > 0;
 }
+
+export const isLokiQuery = (query: DataQuery): query is LokiQuery => {
+  if (!query) {
+    return false;
+  }
+
+  const isLokiQuery = query as LokiQuery;
```

Comment:
Minor, but the fact that the variable is prefixed with `is` makes it look like it's a boolean, when it's actually a query.

Reply:
The naming here comes from how typescript recommends naming type guards: https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards

Reply:
I'm talking about the variable, not the function name. `const isLokiQuery = query as LokiQuery;` It becomes more evident in `return isLokiQuery.expr !== undefined;`

Reply:
Aaa make sense! Fixed in https://github.com/grafana/grafana/pull/66382/files/a8d77694c658214ba4b84e5e0b1dd9b3df7718ca..8aed166bffb94434120f8a2eb850bb6db754d048. 🙂